scripts/jest.js

```diff
@@ -1,8 +1,9 @@
 const jest = require('jest');
-const config = require('./ui_framework_test_config');
+const config = require('../test/jest/config');
```

Comment:
To me this makes complete sense (we've got a script for every-day use, the grunt task "proxies" to that script, and the config is specified in the test folder). I can move them around of course, but I to me this feels like the right approach.

Maybe @epixa has opinions on this? (as he was the one mentioned the `scripts` folder)